### ISSUE TITLE
Crash - Missing zero argument constructor

### ISSUE THREAD

**Issue by ouchadam** [vector-im/element-android#5540]
There's a crash in the verifcation bottom sheet, caused by: 
```
java.lang.InstantiationException: java.lang.Class<im.vector.app.features.crypto.verification.VerificationBottomSheetViewState> has no zero argument constructor
```

```
Thread: main, Exception: java.lang.IllegalStateException: Failed to create initial state!
at com.airbnb.mvrx.RealMavericksStateFactory.createInitialState(MavericksStateFactory.kt:39)
at com.airbnb.mvrx.MavericksFactory.create(MavericksFactory.kt:7)
at com.airbnb.mvrx.MavericksViewModelProvider.get$default(MavericksViewModelProvider.kt:34)
at im.vector.app.features.crypto.verification.VerificationBottomSheet$special$$inlined$fragmentViewModel$default$1.invoke(MavericksExtensions.kt:10)
at im.vector.app.features.crypto.verification.VerificationBottomSheet$special$$inlined$fragmentViewModel$default$1.invoke(MavericksExtensions.kt:1)
at com.airbnb.mvrx.DefaultViewModelDelegateFactory$createLazyViewModel$1.invoke(ViewModelDelegateProvider.kt:2)
at com.airbnb.mvrx.DefaultViewModelDelegateFactory$createLazyViewModel$1.invoke(ViewModelDelegateProvider.kt:1)
at com.airbnb.mvrx.lifecycleAwareLazy.getValue(lifecycleAwareLazy.kt:5)
at com.airbnb.mvrx.lifecycleAwareLazy$initializeWhenCreated$1.onCreate(lifecycleAwareLazy.kt:1)
at androidx.lifecycle.FullLifecycleObserverAdapter.onStateChanged(FullLifecycleObserverAdapter.java:8)
at androidx.lifecycle.LifecycleRegistry$ObserverWithState.dispatchEvent(LifecycleRegistry.java:3)
at androidx.lifecycle.LifecycleRegistry.sync(LifecycleRegistry.java:45)
at androidx.lifecycle.LifecycleRegistry.moveToState(LifecycleRegistry.java:5)
at androidx.lifecycle.LifecycleRegistry.handleLifecycleEvent(LifecycleRegistry.java:2)
at androidx.fragment.app.Fragment.performCreate(Fragment.java:9)
at androidx.fragment.app.FragmentStateManager.create(FragmentStateManager.java:5)
at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:28)
at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:122)
at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:10)
at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:14)
at androidx.fragment.app.FragmentManager$4.run(FragmentManager.java:1)
at android.os.Handler.handleCallback(Handler.java:938)
at android.os.Handler.dispatchMessage(Handler.java:99)
at android.os.Looper.loopOnce(Looper.java:201)
at android.os.Looper.loop(Looper.java:288)
at android.app.ActivityThread.main(ActivityThread.java:7839)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)
Caused by: java.lang.InstantiationException: java.lang.Class<im.vector.app.features.crypto.verification.VerificationBottomSheetViewState> has no zero argument constructor
at java.lang.Class.newInstance(Native Method)
at com.airbnb.mvrx.RealMavericksStateFactory.createInitialState(MavericksStateFactory.kt:23)
```

**Comment by ouchadam** [comment]
looks to be caused by tapping the verification container within the timeline instead of the buttons

| GIF | 
| --- |
![crash](https://user-images.githubusercontent.com/1848238/158630861-a6211459-ea00-44f0-947f-42720ab938f8.gif)

**Comment by ouchadam** [comment]
Fixed in 1.4.6
